extensions/modules/file/src/test/resources/util/fixtures.xqm

```diff
@@ -23,17 +23,85 @@ xquery version "3.1";
 
 module namespace fixtures="http://exist-db.org/xquery/test/util/fixtures";
 
-(: file contents :)
+(: simple xml :)
 
 declare variable $fixtures:XML := document {<foo><bar/></foo>};
+
+declare variable $fixtures:SIMPLE_XML_INDENTED :=
+"<foo>" || $fixtures:NL ||
+"    <bar/>" || $fixtures:NL ||
+"</foo>"
+;
+
+declare variable $fixtures:SIMPLE_XML_UNINDENTED := "<foo><bar/></foo>";
+
+(: more complex xml :)
+
+declare variable $fixtures:COMPLEX_XML :=
+    <root>
+        <item>This is a very long line. Certainly longer than eighty characters. It is here to see what happens to lines longer than a certain limit. Let's see.</item>
+        <empty-item></empty-item>
+        <unary />
+        <item>
+
+            MIXED CONTENT&#10;
+            <nested
+                attr = "with lots of whitespace"
+            />
+            The next word <hi>is</hi> highlighted.
+        </item>
+        <p/>
+    </root>
+;
+
+(: FIXME(JL) cannot use StringConstructor here because that will cause all comparisons to fail on Windows :)
```

Comment:
Could you open a GitHub issue for this and then place the link here so that we link the two things please?